rules_ruby_gapic/ruby_gapic.bzl

```diff
@@ -100,11 +105,14 @@ def ruby_cloud_gapic_library(
   if ruby_cloud_description:
     extra_protoc_parameters.append("ruby-cloud-description={value}".format(value = ruby_cloud_description))
   
-  # Add the cloud form of the generator parameter here. We then pass None to
-  # _ruby_gapic_library_internal so that it doesn't also add the generic form
-  # of the parameter.
+  # Add the cloud form of these generator parameters here. We then pass None to
+  # _ruby_gapic_library_internal so that it doesn't also add the generic forms
+  # of the parameters.
```

Comment:
Consider adding a check that we don't double-specify the "transport" above where the "ruby-cloud-title" and description are checked.

Reply:
We can't yet because compute currently sets the Ruby version of the parameter directly. Will file an issue to clean it up later.